### PR TITLE
Check that credential.id is base64-encoding of credential.rawId

### PR DESCRIPTION
**Pull request by dontcallmedom**
Per https://www.w3.org/TR/webauthn-2/#iface-pkcredential

### CODE REVIEW

File: webauthn/helpers.js [hunk 1]
```diff
@@ -505,6 +514,9 @@ function validatePublicKeyCredential(cred) {
     // rawId
     assert_idl_attribute(cred, "rawId", "should return PublicKeyCredential with rawId attribute");
     assert_readonly(cred, "rawId", "should return PublicKeyCredential with readonly rawId attribute");
+    rawId = cred.rawId;
```
Comment (nsatragno) [inline webauthn/helpers.js:517]: Looks like `rawId` isn't being used.
```suggestion
```

**Review by nsatragno — APPROVED** [review]
Approved with a comment. Good catch, thanks.

**Comment by dontcallmedom** [comment]
@web-platform-tests/wpt-core-team I think this is failing due to #31499 - can you override the CI failure?

File: webauthn/helpers.js (hunk 1 above)
Comment (nsatragno) [reply webauthn/helpers.js:517]: Did you forget about this change?

**Pull request merged**